other/maximum_subarray.py

```diff
@@ -1,4 +1,4 @@
-def max_subarray(nums: list[int]) -> int:
+def max_subarray(nums: list) -> int:
```

Comment:
What is the point of this change?

Reply:
its getting type error with that and not running, so I just removed : 
Traceback (most recent call last):
  File "maxsubbarray.py", line 1, in <module>
    def max_subarray(nums: list[int]) -> int:
TypeError: 'type' object is not subscriptable

Reply:
> its getting Typeerror with that and not running, so I just removed (try it) : Traceback (most recent call last): File "maxsubbarray.py", line 1, in def max_subarray(nums: list[int]) -> int: TypeError: 'type' object is not subscriptable

That is a problem with your python version. `list` being used for typehinting is only available on python versions after python `3.10`

Reply:
oh ok, I just removed the n in my commit then, I kept the type-hinting